### PR TITLE
 DGS-219:basics for subsidie steps

### PR DESCRIPTION
**Pull request by ZijneDwaasheid**
## ID
 DGS-219

 ## Description

making a page where we can see the basic information of the application steps of a subsidy serie

 ## Type of change

 - [ ] Bug fix
 - [x] New feature
 - [ ] Breaking change
 - [ ] Other

 ## How to test

 Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

### CODE REVIEW

File: app/templates/subsidy/application-steps.hbs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                <AuHeading @skin="3">{{step.subsidyProceduralStep.description}}</AuHeading>
+                {{#if step.formSpecification.filename}}
+                  <AuHelpText @skin="secondary" class="au-u-margin-top-none">
+                    {{#if (or step.deadline.begin step.deadline.end)}}
+                      {{moment-format step.deadline.begin "DD MMMM YYYY"}}
+                      &#8211;
+                      {{moment-format step.deadline.end "DD MMMM YYYY"}}
+                    {{else}}
+                      N.V.T.
+                    {{/if}}
```
Comment (wolfderechter) [inline app/templates/subsidy/application-steps.hbs]:
```suggestion
                    {{moment-format step.deadline.begin "DD MMMM YYYY"}}
                    &#8211;
                    {{moment-format step.deadline.end "DD MMMM YYYY"}}
```
Every step should have start and end date so not needed here

**Review by wolfderechter — COMMENTED** [review]
I think I would keep the order the same, so starting with 0

**Comment by ZijneDwaasheid** [comment]
> I think I would keep the order the same, so starting with 0

alright

File: app/templates/subsidy/measure-offer.hbs [hunk 2]
```diff
@@ -43,13 +43,7 @@
...
                 @model={{series.id}}
               >
-                {{#if series.title}}
-                  {{series.title}}
-                {{else}}
-                  <AuHelpText>
-                    Geen titel
-                  </AuHelpText>
-                {{/if}} 
+                {{series.label}}
```
Comment (wolfderechter) [inline app/templates/subsidy/measure-offer.hbs:46]: @ZijneDwaasheid I changed this to use the label function, which checks for title and fallbacks to create title based on start/end date of period. It's also used in loket so best to mimic closely

**Pull request merged**